pallets/native-token-management/src/lib.rs

```diff
@@ -138,6 +141,13 @@ pub mod pallet {
 			token_amount: NativeTokenAmount,
 		) -> DispatchResult {
 			ensure_none(origin)?;
+			Initialized::<T>::mutate(|initialized| {
+				if !*initialized {
+					log::info!("");
+					*initialized = true
+				}
+				true
+			});
```

Comment:
Actually, cost of doing it here with db-sync follower isn't very big, the query if fast, because it first gets transactions by address and asset and only then filter outs by block. IDK how it will be with other followers.

So, it is your decision if you try to move it somewhere else.

Reply:
I will leave this as-is for now. The cost will be nullified by batching in the future if it is necessary to do so.